src/python/pants/backend/native/subsystems/conan.py

```diff
@@ -49,6 +50,10 @@ class Conan(Subsystem):
   def implementation_version(cls):
     return super(Conan, cls).implementation_version() + [('Conan', 0)]
 
+  @classmethod
+  def subsystem_dependencies(cls):
+    return super(Conan, cls).subsystem_dependencies() + (PythonRepos, PythonSetup)
```

Comment:
How are these used here?

Reply:
`pants.backend.python.tasks.pex_build_util.dump_requirements` sneakily grabs the global instances without requesting them, so you were getting lucky here. See https://github.com/pantsbuild/pants/pull/6202#discussion_r203904315 for some discussion of thi problem. Nora is working on a fix.